### DISCUSSION TITLE
How to save select2 multiple to antoher table request

### DISCUSSION THREAD

**Discussion by rickyviz** [Laravel-Backpack/community-forum#281] [Q&A (Help)]
i have 1 formwhere include same field not define in one table,how to save select2 mutliple data to the relation

**Comment by jcastroa87** [comment]
Hello @rickyviz im sorry for soo late answer, but a simple way here is using an observer.

You can follow this [tutorial](https://raviyatechnical.medium.com/laravel-basic-how-to-use-laravel-model-observers-857a4ba79f81) 

Cheers.
